ci-operator/jobs/openshift/cluster-api/OWNERS

```diff
@@ -1,12 +1,12 @@
-# DO NOT EDIT; this file is auto-generated using tools/populate-owners.
-# from https://github.com/openshift/cluster-api/blob/bd4f0ba460758c33c609cb333721606c6d002c4e/OWNERS
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
 - bison
 - enxebre
 - frobware
 - ingvagabund
+- paulfantom
```

Comment:
It looks correct to me.
https://github.com/openshift/cluster-api/blob/4024de4fa64d3f88794cff66b85c53fec176d6a6/OWNERS#L8